firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/FirebaseAppDistributionRegistrar.java

```diff
@@ -57,14 +58,18 @@ private FirebaseAppDistribution buildFirebaseAppDistribution(ComponentContainer
     FirebaseAppDistributionLifecycleNotifier lifecycleNotifier =
         FirebaseAppDistributionLifecycleNotifier.getInstance();
 
-    if (firebaseApp.getApplicationContext() instanceof Application) {
-      Application firebaseApplication = (Application) firebaseApp.getApplicationContext();
+    Context context = firebaseApp.getApplicationContext();
+    if (context instanceof Application) {
+      Application firebaseApplication = (Application) context;
       firebaseApplication.registerActivityLifecycleCallbacks(lifecycleNotifier);
     } else {
       LogWrapper.getInstance()
           .e(
               TAG
-                  + "Error registering app to ActivityLifecycleCallbacks. SDK might not function correctly.");
+                  + "Context "
+                  + context
+                  + " was not an application, can't register for lifecycle callbacks. SDK might not"
```

Comment:
Updated this to capitalize "Application." While it sounds awkward it's correct since the Context object is expected to be an instance of Application. For the record I copied this from another SDK.

Reply:
That is fine then. the only reason this is confusing for me is because although it says instance of application based off of what I have read the context itself isn't an application. This is more of a nit than anything but this is one of the sources that I was reading. https://blog.mindorks.com/understanding-context-in-android-application-330913e32514

Reply:
Ah, thanks for the reference. Reading that, I do get the impression that the context either "is" an application, or "is" an activity. For example: "Application Context: It is the application and we are present in Application. For example - MyApplication(which extends Application class). It is an instance of MyApplication only."

Reply:
That makes sense. Another resource I looked at discussed context as a base class that application and other parts are built off of so that makes more sense that context is an application. My confusion stemmed from some places were saying the context comes from the application which is true but that made me think of it as a part of the application. Thanks for helping clarify that.